vent/controller/control_module.py

```diff
@@ -56,6 +57,8 @@ def __init__(self, save_logs: bool = False, flush_every: int = 10):
         self.logger = init_logger(__name__)
         self.logger.info('controller init')
 
+        prefs.set_time()  # Keep track of when the control_module has been started the first time.
```

Comment:
As discussed, this marks the time when the controller was started for the first time

Reply:
see my comments in prefs, it's a minor thing but i think something better handled there

Reply:
good suggestion. Fixed